docs/product/sentry-basics/distributed-tracing/initialize-sentry-sdk-frontend.mdx

```diff
@@ -0,0 +1,126 @@
+   yarn add @sentry/react
+   ```
+
+   ```bash {tabTitle:pnpm}
+   pnpm add @sentry/react
+   ```
+
+1. Import and initialize the SDK.
+
+   Open `src/index.js` and add the following lines of code below the last import statement:
+
+   ```javascript {filename:src/index.js}
+   import * as Sentry from "@sentry/react";
+
+   Sentry.init({
+      dsn: "your_DSN_key",
+      integrations: [
+               Sentry.browserTracingIntegration(),
+               Sentry.replayIntegration(),
+            ],
+      // Tracing
+      tracesSampleRate: 1.0, //  Capture 100% of the transactions
+      // Set 'tracePropagationTargets' to control for which URLs distributed tracing should be enabled
+      tracePropagationTargets: ["localhost", /^https:\/\/yourserver\.io\/api/],
+      // Session Replay
+      replaysSessionSampleRate: 0.1, // This sets the sample rate at 10%. You may want to change it to 100% while in development and then sample at a lower rate in production.
+      replaysOnErrorSampleRate: 1.0, // If you're not already sampling the entire session, change the sample rate to 100% when sampling sessions where errors occur.
+   });
+   ```
+
+   It's important to import and initialize the SDK as early as possible in your application's lifecycle so Sentry can capture errors throughout the lifecycle.
+
+1. Add your DSN key to the Sentry SDK configuration.
+
+   Paste in the DSN key value you copied from the project created in the [previous section](/product/sentry-basics/distributed-tracing/create-new-project/).
+
+   ```javascript {filename:src/index.js}
+   Sentry.init({
+     dsn: "<your_DSN_key>",
+     // ...
+   });
+   ```
+
+1. Save the file.
+
+The options set in `Sentry.init()` are called the SDK's configuration. The only required configuration option is the DSN. However, the SDK supports many other configuration options. Learn more in our [Configuration](/platforms/javascript/guides/react/configuration/) docs.
+
+The configuration above enables Sentry's error monitoring feature, as well as its [Performance](/platforms/javascript/guides/react/performance) (tracing) and [Session Replay](/platforms/javascript/guides/react/session-replay) features.
+
+## 3. Build and Run the Sample Application
+
+In the `tracing-tutorial-frontend` project folder:
+
+1. Install project dependencies.
+
+   ```bash
+   npm install
+   ```
+
+1. Start the application in develop mode.
+
+   ```bash
+   npm start
+   ```
+
+   Once the application starts, you'll see a confirmation message similar to this one in your terminal:
+
+   ```bash
+   Compiled successfully!
+
+   You can now view tracing-demo in the browser.
+
+      Local:            http://localhost:3000
+      On Your Network:  http://10.0.0.39:3000
+   ```
+
+   > **Troubleshooting tip**: If the application fails to start due to syntax errors or errors for missing dependencies/modules, make sure you're using Node 18+ and install dependencies again. Run `nvm use 18` and then `npm install`.
```

Comment:
This is taken from the 'frontend tutorial' and I _think_ `nvm` is correct here, referring to 'Node Version Manager' which lets users switch between node versions using `nvm use X`. But maybe that should be more explicit since not every developer will be familiar with nvm. 

Would it be worth the extra real estate to be a little more verbose here, like:
```
...install dependencies again. You can run `node -v` to see your node version, and if you are using Node Version manager you can run `nvm use 18` to switch to version 18.
```

Reply:
Ah ok, got it!